src/data/set/pointwise.lean

```diff
@@ -450,23 +460,26 @@ localized "attribute [instance] set.comm_monoid set.add_comm_monoid" in pointwis
 
 open_locale pointwise
 
-/-- Repeated pointwise addition (not the same as pointwise repeated addition!) of a `finset`. -/
+/-- Repeated pointwise addition (not the same as pointwise repeated addition!) of a `finset`. See
+Note [pointwise nat action].-/
 protected def has_nsmul [has_zero α] [has_add α] : has_scalar ℕ (set α) := ⟨nsmul_rec⟩
```

Comment:
Did you reorder these instances in a previous commit? I would have expected this one to come before `set.monoid` so that it could be used as the `nsmul` field.

Reply:
You're right I messed up somewhere. Is that better now?